### PR TITLE
Store logic

### PR DESCRIPTION
**Pull request by metcalf-putnam**
Beginnings of store, with logic changes to how days and money are handled, making source of truth GameLogic.

### CODE REVIEW

**Comment by metcalf-putnam** [comment]
Still need to hook up game logic to update store with quantities when items are purchased, and, you know, make the store a place you can access on the day end screen.

File: game_utils/game_logic.gd [hunk 1]
```diff
@@ -2,6 +2,11 @@ extends Node
 var customers = ["young_man"]
 var weights = [10]
 var prob_dist = [1]
+var day := 1
+var money := 67 setget set_player_money, get_player_money
```
Comment (influffity) [inline game_utils/game_logic.gd:6]: Is 67 intentional?

File: interfaces/store/store_item.gd [hunk 2]
```diff
@@ -1,38 +1,64 @@
...
-  pass
-  #EventHub.emit_signal("interactable_broadcasted", description)
+  $Owned/Amount.bbcode_text = str(owned)
+  $Price/Amount.text = "$" + str(price)
+
+  set_button()
+
+
+func check_can_purchase(funds):
+  if funds < price or stocked_items <= 0:
```
Comment (influffity) [inline interfaces/store/store_item.gd:41]: Why not
`$Button.disabled =  funds < price or stocked_items <= 0`

File: models/Ticket/sprite.png.import [hunk 3]
```diff
@@ -2,15 +2,15 @@
...
+path="res://.import/sprite.png-fc96854d5d20c38d1d3c0aa8b3b7aa88.stex"
 metadata={
 "vram_texture": false
 }
 
 [deps]
 
-source_file="res://models/Ticket/sprite.png"
-dest_files=[ "res://.import/sprite.png-76c28b1d43124eab8d44c32598cddea5.stex" ]
+source_file="res://models/ticket/sprite.png"
```
Comment (influffity) [inline models/Ticket/sprite.png.import:12]: yaaay Ticket -> ticket is good puppeh kitteh

**Review by influffity — APPROVED** [review]

**Pull request merged**